Library/Formula/procmail.rb

```diff
@@ -0,0 +1,78 @@
+class Procmail < Formula
+  desc "Procmail can be used to create mail-servers, mailing lists, sort your incoming mail into separate folders/files"
```

Comment:
Please use a shorter description. “procmail: <description>” should fit under 80 characters.